static/src/javascripts/projects/common/modules/identity/sign-in-gate/helper.js

```diff
@@ -103,6 +114,8 @@ export const isInvalidArticleType = (include: Array<string> = []): boolean => {
         'isPhotoEssay',
         'isSensitive',
         'isSplash',
+        'isPodcast',
+        'isInvalidArticleType',
```

Comment:
what does this refer too?
I don't see this referred to in the `window.guardian.config.page` object?

Reply:
fixed.